modules/postman-templating/src/errors.ts

```diff
@@ -4,4 +4,4 @@ export class TemplateError extends Error {
     Object.setPrototypeOf(this, new.target.prototype) // restore prototype chain
     Error.captureStackTrace(this)
   }
-}
```

Comment:
This is a little weird; what changes are made to this file?

Reply:
just linted it, the ⛔️  means there no new line at the end of file